frame/timestamp/src/lib.rs

```diff
@@ -254,7 +254,7 @@ impl<T: Config> Pallet<T> {
 	}
 
 	/// Set the timestamp to something in particular. Only used for tests.
-	#[cfg(feature = "std")]
+	#[cfg(any(feature = "runtime-benchmarks", test))]
```

Comment:
staking use this in its test.

That means now staking tests can only be executed with the feature runtime-benchmarks. I would prefer to keep std
```suggestion
	#[cfg(any(feature = "runtime-benchmarks", feature = "std"))]
```

Reply:
Doesn't the `test` part covers it?

Reply:
`test` is not a feature, it is only activated when running the test on the crate itself. it is not activated for the dependencies

Reply:
Right. I 100% falls into this trap before but I guess I will never know how many more times I am going to fall into the same one in future...